### ISSUE TITLE
Neither Open Shell 4.4.160 or Classic Shell 4.3.1 will install or open on fully patched to Jan 2021 (ESU) Windows 7 Pro x86

### ISSUE THREAD

**Issue by Clinical-Metrics-LLC** [Open-Shell/Open-Shell-Menu#639]
Problem is that neither Open Shell 4.4.160 or Classic Shell 4.3.1 will install or operate with with fully patched (to Jan 2021 (ESU)) Windows 7 Pro x86.

**To Reproduce**
1. When Windows 7 Pro x86 is updated to KB-4534310  (Jan 2020), both Open Shell and Classic Shell install normally and operate normally,
2. When updated through probably KB-4539602 (Feb 2020 - Screen Blackout Fix)) and perhaps KB-4555449 (May 2020 - Servicing Stack Update), previously installed Open Shell or Classic Shell causes Windows Explorer to hand and Open/Classic Shell do no not operate.  Windows Explorer repeatedly hangs, crashes and restarts.  Opening Task Manager shows explore.exe to chew up more and more memory (1 GB + out of 2 GB RAM) until in crashes, then starts at several 10 K and repeats the cycle.
3.  When update as in 2., in attempting to install, UAC asks for permission to load, then nothing happens (no install happens)
4. With full ESU updates to  KB-4592471 Dec 2020 (Security Monthly Roll-up) and KB-4592503 Dec 2020 (Security only update), same behavior as in 2. and 3. occurs (in other words, what ever happened early was not fixed in subsequent rollups).

**Expected behavior**
Cleanly installation, if installed, it works

**Screenshots**
I
**Version:**
 - Open Shell 4.4.160
 - Classic Shell 4.3.1
 - 
 - OS: Windows 7 Pro x86 through last "official" update to Jan 2020 - last patch KB-4534310  (Jan 2020 - Security Monthly Quality Rollup) 
 - OS: Patches that may have broken this:  KB-4539602 (Feb 2020 -Black Screen Fix ), KB-4555449 May 2020 - Servicing Stack Update)
 - OS: Windows 7 Pro x86 - ESU patched through KB-4592471 Dec 2020 (Security Monthly Roll-up) and KB-4592503 Dec 2020 (Security only update)

**Additional context**
System used is an old ThinkPadT-42 with AMD 9600 M-10 GPU with 2 GB RAM and 128 GB PATA Solid State.  System is on its last load after being out of service with patching through roughly 2018.  Classic Shell work fine back then.

Other programs that "broke" with this are SnagIt 10.0.2 (refuses to install like Open/Classic Shell), Kaspersky Internet Security 21.2.16.590 loads with an error "Error 27300. Error installing driver klim6.sys_X86. Error code: -2147024888. Changes made in the operating system will be automatically rolled back."

**Comment by NebelNidas** [comment]
There was an issue with Windows 7 which has been fixed in b2df22104ee04e3f926628483a39f77a1ba3e7fd. but I think it was only related to the installer and has nothing to do with your issue.

Nevertheless, could you test with the latest nightly build anyway? Perhaps there were some other fixes I missed. https://ci.appveyor.com/project/passionate-coder/open-shell-menu/branch/master/artifacts

**Comment by Clinical-Metrics-LLC** [comment]
I tried to install 4.4.163.  It does pretty much the same thing.  I did it as an Administrative install.  The AUC message appears then gives a "hourglass" for a second or two, then nothing.  BTW - I pre-emptively installed all available, C++ runtimes from 2005-2019 before I installed anything, including Open Shell.

I have not gone back and installed prior to the ESU updates, but suspect the install problem has to be solved first.  Also (prob not relevant here the GPU was branded as an ATI, from before AMD took them over).

And thanks for looking into this - if there is anything else I can do to help you, let me know.
